src/auth_required.js

```diff
@@ -1,3 +1,7 @@
 /***
  * @todo Redirect the user to login page if token is not present.
  */
+
+if (localStorage.token === undefined) {
+    window.location.href = 'login/';
+} 
```

Comment:
Try to work with functions and import them, as those helper functions would be more helpful in case you need to call them multiple times in your code.